tests/test_postgresql.py

```diff
@@ -127,20 +128,22 @@ def setUp(self) -> None:
         self.table = table(self.table_name)
 
     def test_special_char_password(self):
+        username = "test"
         password = "passw!!!@rd"
         # Setup user with special character '@' in password
-        self.connection.query("DROP USER IF EXISTS test;", None)
-        self.connection.query(f"CREATE USER test WITH PASSWORD '{password}';", None)
+        self.connection.query(f"DROP USER IF EXISTS {username};", None)
+        self.connection.query(f"CREATE USER {username} WITH PASSWORD '{password}';", None)
 
         password_quoted = quote(password)
-        db_config = {
-            "driver": "postgresql",
-            "host": "localhost",
-            "port": 5432,
-            "dbname": "postgres",
-            "user": "test",
-            "password": password_quoted,
-        }
+        db_config = deepcopy(self.connection._args)
+        db_config.update(
```

Comment:
Make this config explicit

db_config = {
            "driver": "postgresql",
            "host": "localhost",
            "port": 5432,
            "dbname": "postgres",
            "user": username,
            "password": password_quoted,
        }

Reply:
if explicit this test fails since it is not honouring configuration in `TEST_POSTGRESQL_CONN_STRING`.

Reply:
ahh fair callout. I'll resolve this then.